scripts/utils/internet_detect_tool.py

```diff
@@ -0,0 +1,246 @@
+from PyQt5.QtWidgets import QApplication, QMenu
+from PyQt5.QtCore import QTimer, QThread, pyqtSignal
+import threading
+import requests
+import urllib3
+import win32api
+import win32gui
+import win32con
+import winreg
+import struct
+import signal
+import ctypes
+import time
+import os
+
+
+# Define constants
+CHECK_INTERVAL = 2  # Seconds
+CONNECT_TEST_URL = "https://www.msftconnecttest.com/connecttest.txt"
+EXPECTED_RESPONSE = "Microsoft Connect Test"
+SPI_SETDESKWALLPAPER = 20
+SPIF_UPDATEINIFILE = 0x01
+SPIF_SENDWININICHANGE = 0x02
+COLOR_DESKTOP = 1
+ICON_RED_X = "red_circle.ico"
+ICON_FLARE = "flare.ico"
```

Comment:
@mandiant/commando-vm is there interest in commando VM to use a tool like this? If so, we should keep it generic.